_resources/port1.0/group/python-1.0.tcl

```diff
@@ -273,7 +309,7 @@ proc python_get_defaults {var} {
             }
         }
         lib {
-            if {${python.version} != 24 && ${python.version} != 25} {
+            if {${python.version} != 24 && ${python.version} != 25 && ![string match py* ${python.version}]} {
```

Comment:
I don't recall exactly where pypy's libpython lives, but this doesn't look correct.